doc/connectivity/bluetooth/autopts/autopts-linux.rst

```diff
@@ -92,29 +120,31 @@ If you type 'ifconfig' in terminal, you should be able to find your host IP:
    :align: center
 
 VirtualBox
-^^^^^^^^^^^^^
+----------
+
+VirtualBox on Linux, macOS and Solaris Oracle VM VirtualBox will only allow IP addresses in
+``192.168.56.0/21`` range to be assigned to host-only adapters, so if using a static address with
```

Comment:
One thing to mention here is that the address in the VMWare and Windows images are using 192.168.2.1 and 192.168.2.2.

I don't have VMWare to update those screenshots; is it clear enough that 192.168.56* needs to be used instead of 192.168.2* for VirtualBox?

Reply:
Seems to be clear enough to me